static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -66,18 +68,21 @@ function MessageFormatter({breadcrumb}: MessageFormatterProps) {
 
 interface ConsoleMessageProps extends MessageFormatterProps {
   isLast: boolean;
-  relativeTimestamp: string;
+  startTimestamp: number;
 }
-function ConsoleMessage({
-  breadcrumb,
-  isLast,
-  relativeTimestamp = '',
-}: ConsoleMessageProps) {
+function ConsoleMessage({breadcrumb, isLast, startTimestamp = 0}: ConsoleMessageProps) {
   const ICONS = {
     error: <IconClose isCircled size="xs" />,
     warning: <IconWarning size="xs" />,
   };
 
+  const {setCurrentTime, setCurrentHoverTime} = useReplayContext();
+
+  const diff = relativeTimeInMs(breadcrumb.timestamp || '', startTimestamp);
+  const onClick = () => setCurrentTime(diff);
+  const onMouseOver = () => setCurrentHoverTime(diff);
+  const onMouseOut = () => setCurrentHoverTime(undefined);
```

Comment:
Small nit, can we name these with a `handle` prefix? https://develop.sentry.dev/frontend/#event-handlers